src/sentry/models/organizationmembermapping.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.db.models import BoundedBigIntegerField, FlexibleForeignKey, Model, sane_repr
+from sentry.db.models.base import control_silo_only_model
+from sentry.models.organizationmember import InviteStatus
+from sentry.roles import organization_roles
+
+
+@control_silo_only_model
+class OrganizationMemberMapping(Model):
+    """
+    This model resides exclusively in the control silo, and will
+    - map a user or an email to a specific organization to indicate an organization membership
+    """
+
+    __include_in_export__ = True
+
+    organization_id = BoundedBigIntegerField(db_index=True)
```

Comment:
We should use `organization_id = HybridCloudForeignKey("sentry.Organization", on_delete="CASCADE")` here I think.   HybridCloudForeignKeys help our cross silo deletions framework perform async deletion processing.  In this case, you won't need to do anything else (Organization delete cascading is already setup!).